src/coreclr/jit/lowerxarch.cpp

```diff
@@ -2506,8 +2622,65 @@ GenTree* Lowering::LowerHWIntrinsicCreate(GenTreeHWIntrinsic* node)
     //          +--*  ... T
     //          +--*  opN T
     //   node = *  HWINTRINSIC   simd   T Create
+    if (intrinsicId == NI_Vector512_Create)
+    {
+        assert(comp->compIsaSupportedDebugOnly(InstructionSet_AVX2));
```

Comment:
This one should also be `IsBaselineVector512IsaSupportedDebugOnly()` since we generate `InsertVector256`

Reply:
Done